packages/client/screens/UserProfile/index.tsx

```diff
@@ -25,13 +24,12 @@ const UserProfileScreen: React.FC<UserProfileScreenProps> = ({
         () =>
             createUserProfileInformationMachine({
                 userID,
-                config: getUserProfileInformationMachineOptions(),
             }),
         [userID],
     );
 
     const userProfileInformationService = useInterpret(
-        userProfileInformationMachineConfigured,
+        () => userProfileInformationMachineConfigured,
     );
```

Comment:
As for `packages/client/screens/MyProfile/index.tsx` file, providing a function to `useInterpret` serves the same purpose as using a `useMemo` to compute the state machine only once.

```suggestion
    const userProfileInformationService = useInterpret(
        () => createUserProfileInformationMachine({ userID }),
    );
```

Reply:
Isn't it a problem for the userID to do not be watched in the useMemo ?